native/sdk/src/pallet/spaces.rs

```diff
@@ -22,6 +22,7 @@ pub trait Spaces: System + Balances {
         + Send
         + Sync
         + 'static;
+    type ScoringAction: Default + Decode + Encode + Send + Sync + 'static;
```

Comment:
Please add TODO, that scores should be deleted everywhere, after we remove them from blockchain